lib/screens/projects_screen.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:flutter/material.dart';
+import 'package:protofast/data/projects_manager.dart';
+import 'package:protofast/models/project.dart';
+import 'package:protofast/screens/new_project_screen.dart';
+import 'package:protofast/screens/project_screen.dart';
+
+import 'package:protofast/features/user_login.dart';
+import 'package:protofast/features/user_profile.dart';
+
+class ProjectsScreen extends StatefulWidget {
+  const ProjectsScreen({Key? key}) : super(key: key);
+
+  @override
+  State<ProjectsScreen> createState() => _ProjectsScreenState();
+}
+
+class _ProjectsScreenState extends State<ProjectsScreen> {
+  List<Project> projects = [];
+
+  Future<void> loadProjects() async {
+    projects = await ProjectsManager.loadProjects();
+    Project sampleProject = Project('WeTube', [UserLogin(), UserProfile()],
+        [Platforms.android, Platforms.web]);
+    projects.add(sampleProject);
+    setState(() {});
+  }
+
+  @override
+  void initState() {
+    loadProjects();
```

Comment:
It looks like a bug here, because when it loads projects only once the application starts first time. When returned back from other screens it seems not working. Could you please fix it?

Reply:
I will fix in my branch, so no worries